### ISSUE TITLE
support generate bash completion 

### ISSUE THREAD

**Issue by wu0407** [ahmetb/kubectl-tree#13]
like this `kubectl tree completion bash`

**Comment by ahmetb** [comment]
Plugins don’t support bash completion. See Kubernetes repo for relevant issue.

**Comment by ahmetb** [comment]
Correct. I shall open this issue. We're yet to implement in a generic way for Krew plugins. There's an open issue for that in krew repo as well.
